### PR TITLE
Feb 2021 release notes

### PR DESCRIPTION
**Pull request by dannify**

### CODE REVIEW

File: packages/dev/docs/pages/releases/2021-02-16.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: After a small break, we are back with another release. This one is jam packed with fixes and new features. We have improved Picker, Tooltip, Combobox and updated a number of our aria hooks. Also check out our new components currently under construction including a Color package. A special thanks to all of our contributors for a very active new year.
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2021-02-16.mdx:14]: Nit, not sure if we want to have components with backticks, we've had them with and without in previous release notes:
```suggestion
description: After a small break, we are back with another release. This one is jam packed with fixes and new features. We have improved `Picker`, `Tooltip`, `ComboBox` and updated a number of our aria hooks. Also check out our new components currently under construction including a `Color` package. A special thanks to all of our contributors for a very active new year.
```

File: packages/dev/docs/pages/releases/2021-02-16.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+date: 2021-02-16
+---
+
+# February 16, 2021 Release
+
+After a small break, we are back with another release. This one is jam packed with fixes and new features. We have improved Picker, Tooltip, Combobox and updated a number of our aria hooks. Also check out our new components currently under construction including a Color package. A special thanks to all of our contributors for a very active new year.
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2021-02-16.mdx:20]: Same as above
```suggestion
After a small break, we are back with another release. This one is jam packed with fixes and new features. We have improved `Picker`, `Tooltip`, `ComboBox` and updated a number of our aria hooks. Also check out our new components currently under construction including a `Color` package. A special thanks to all of our contributors for a very active new year.
```

File: packages/dev/docs/pages/releases/2021-02-16.mdx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+- Add `order` to `styleProps` - [@renancleyson-dev](https://github.com/renancleyson-dev) - [PR](https://github.com/adobe/react-spectrum/pull/1351)
+- Update component scaffolding - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/1364)
+- Keep `Tooltip` open when hovered - [@Andarist](https://github.com/Andarist) - [PR](https://github.com/adobe/react-spectrum/pull/1377)
+- Support left/right key in `useSelect` - [@tomsontom](https://github.com/tomsontom) - [PR](https://github.com/adobe/react-spectrum/pull/1299)
+- Add filtering to `useAsyncList` hook - [@LFDanLu](https://github.com/LFDanLu) - [Docs](https://react-spectrum.adobe.com/react-stately/useAsyncList.html#filtering)
+- Async support for `ComboBox` - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/1335) - [Docs](https://react-spectrum.adobe.com/react-spectrum/ComboBox.html#asynchronous-loading)
+- Update validation Icon position in `TextArea` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1543)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2021-02-16.mdx:31]: nit:
```suggestion
- Update validation icon position in `TextArea` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1543)
```

File: packages/dev/docs/pages/releases/2021-02-16.mdx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+
+## Fixed
+- Add resize observing to `Picker` after initial render - [@tomsontom](https://github.com/tomsontom) - [PR](https://github.com/adobe/react-spectrum/pull/1284)
+- Add `BreadcrumbItem` type interface - [@stefanprobst](https://github.com/stefanprobst) - [PR](https://github.com/adobe/react-spectrum/pull/1317)
+- Update translation in `Picker` - [@RafalFilipek](https://github.com/RafalFilipek) - [PR](https://github.com/adobe/react-spectrum/pull/1309)
+- Add resize observing to `Combobox` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1333)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2021-02-16.mdx:38]:
```suggestion
- Add resize observing to `ComboBox` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/1333)
```

**Review by snowystinger — APPROVED** [review]
lgtm

File: packages/dev/docs/pages/releases/2021-02-16.mdx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+## Under construction
+
+Pre-release versions of the following components have been released. Please feel free to try them out, and report any issues you encounter.
+
+- NumberField - [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/NumberField.html), [React Aria](https://react-spectrum.adobe.com/react-aria/useNumberField.html)
+- ColorWheel - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorWheel.html)
+- ColorSlider - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorSlider.html)
+- ColorField - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorField.html)
+- ComboBox - [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ComboBox.html), [React Aria](https://react-spectrum.adobe.com/react-aria/useComboBox.html)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2021-02-16.mdx:81]: ComboBox had already been released as an alpha in a previous release, do we want to include it here?

**Review by LFDanLu — COMMENTED** [review]
just some nits and a question

File: packages/dev/docs/pages/releases/2021-02-16.mdx (hunk 5 above)
Comment (dannify) [reply packages/dev/docs/pages/releases/2021-02-16.mdx:81]: yeah we made updates and no harm in calling out again for people to try

File: packages/dev/docs/pages/releases/2021-02-16.mdx (hunk 1 above)
Comment (dannify) [reply packages/dev/docs/pages/releases/2021-02-16.mdx:14]: probably not in the meta description

File: packages/dev/docs/pages/releases/2021-02-16.mdx (hunk 5 above)
Comment (LFDanLu) [reply packages/dev/docs/pages/releases/2021-02-16.mdx:81]: sounds good

**Review by LFDanLu — APPROVED** [review]

**Review by devongovett — APPROVED** [review]

**Pull request merged**